pkg/driver/trace_driver.go

```diff
@@ -74,7 +74,7 @@ func NewDriver(driverConfig *config.Configuration) *Driver {
 // HELPER METHODS
 // ///////////////////////////////////////
 func (d *Driver) outputFilename(name string) string {
-	return fmt.Sprintf("%s_%s_%d.csv", d.Configuration.LoaderConfiguration.OutputPathPrefix, name, d.Configuration.TraceDuration)
+	return fmt.Sprintf("%s_%s_%d_%d.csv", d.Configuration.LoaderConfiguration.OutputPathPrefix, name, d.Configuration.TraceDuration, len(d.Configuration.Functions))
```

Comment:
Why do you need to change the filename format?

Reply:
I changed it because of the automated deployer, which runs a sweep over the number of functions with the same durations. So without this, the same file would be overwritten multiple times.

Reply:
Overall, is the automatic deployer an integral part of this PR? Can the automatic deployer be changed so as not to overwrite the file? I ask this because that messes up the interface with the invitro (which you didn't declare in the PR description BTW) that my analysis code relies on. So, change it only if you actually need it to be different.

Reply:
Will change it back and try to figure something else out for the automatic deployer.